eidpdata/src/main/java/uk/ac/ucl/eidp/data/PoolStrategy.java

```diff
@@ -78,62 +65,69 @@ private void initialiseConnection() {
   }
 
   @Override
-  public List<Map<String, String>> processDbCall(String methodPath, Map<String, String> parameters) {
+  public List<Map<String, String>> processDbCall(String methodId, Map<String, String> parameters) {
     if (null == connection) {
       initialiseConnection();
     }
 
-    String sqlStatement = statementGenerator.getSqlStatement(methodPath);
+    String sqlStatement = statementGenerator.getSqlStatement(methodId);
 
     List<String> fields = getJdbcFields(sqlStatement.split(";")[0]);
     String jdbcStatement = sqlStatement.replaceAll(findParametersPattern.patter‌n(), "?");
 
-    String datasetPath = methodPath.substring(0, methodPath.lastIndexOf('.'));
-    Map<String, Parameter> p = statementGenerator.getParameterSettings(parameters.keySet(), datasetPath);
-    List<Map<String, String>> l = new ArrayList<>();
+    String datasetPath = methodId.substring(0, methodId.lastIndexOf('.'));
+    Map<String, Parameter> parameterMap = statementGenerator.getParameterSettings(
+         parameters.keySet(), 
+         datasetPath
+    );
+    List<Map<String, String>> resultList = new ArrayList<>();
     try {
-      int scroll_type = 1004; // defaults to ResultSet.TYPE_SCROLL_INSENSITIVE
-      int concurrency_mode = 1007; // defaults to ResultSet.CONCUR_READ_ONLY
-      if (properties.containsKey(RS_SCROLL_TYPE)) {
-        scroll_type = Integer.getInteger(properties.getProperty(RS_SCROLL_TYPE));
+      int scrollType = 1004; // defaults to ResultSet.TYPE_SCROLL_INSENSITIVE
+      int concurrencyMode = 1007; // defaults to ResultSet.CONCUR_READ_ONLY
+      if (properties.containsKey(rsScrollType)) {
+        scrollType = Integer.getInteger(properties.getProperty(rsScrollType));
       }
-      if (properties.containsKey(RS_SCROLL_TYPE)) {
-        concurrency_mode = Integer.getInteger(properties.getProperty(RS_CONCURRENCY_MODE));
+      if (properties.containsKey(rsScrollType)) {
+        concurrencyMode = Integer.getInteger(properties.getProperty(rsConcurrencyMode));
       }
-      PreparedStatement ps = connection.prepareStatement(jdbcStatement.split(";")[0], scroll_type, concurrency_mode);
-      setValues(ps, fields, parameters, p);
+      PreparedStatement ps = connection.prepareStatement(
```

Comment:
\[FindBugs\] INFO: OBL: uk\.ac\.ucl\.eidp\.data\.PoolStrategy\.processDbCall\(String, Map\) may fail to clean up java\.sql\.Statement